src/pages/HomePage/index.js

```diff
@@ -9,7 +9,7 @@ import {
 
 import { makeStyles } from '@material-ui/core/styles';
 
-import config from '../../data/home_config.json';
+import { homeConfig } from '../../data/index.json';
```

Comment:
can you use `data.json` instead of `data/index.json` ?

Reply:
In the next commit, we will get data from the Firebase cloud. And we will not use this file